tests/settings.py

```diff
@@ -12,7 +12,6 @@
 # Reconfigure pulse to operate on default vhost of rabbitmq
 PULSE_URI = BROKER_URL
 PULSE_URL = BROKER_URL
-PULSE_EXCHANGE_NAMESPACE = 'test'
```

Comment:
I guess this is fine so long as the tests are guaranteed to never inherit any locally set pulse credentials - which they shouldn't so long as no one removes the lines above this one :-)